manifests/03_credentials_request_gcp.yaml

```diff
@@ -17,12 +17,39 @@ spec:
   providerSpec:
     apiVersion: cloudcredential.openshift.io/v1
     kind: GCPProviderSpec
-    predefinedRoles:
-      # FIXME: find a replacement for instanceAdmin, since the CSI driver
-      # only needs "compute.instances.[get|attachDisk|DetachDisk]"
-      - roles/compute.instanceAdmin
-      - roles/compute.storageAdmin
-      - roles/iam.serviceAccountUser
+    # Required driver permissions: https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/blob/1a1f846c41c963e17b4757ce3beb0bf1e817d473/docs/kubernetes/user-guides/driver-install.md?plain=1#L17-L21
+    permissions:
+      - "compute.instances.get"
+      - "compute.instances.attachDisk"
+      - "compute.instances.detachDisk"
+      - "compute.diskTypes.*"
+      - "compute.disks.*"
+      - "compute.globalOperations.get"
+      - "compute.globalOperations.list"
+      - "compute.images.*"
+      - "compute.instanceSettings.get"
+      - "compute.instantSnapshots.*"
+      - "compute.licenseCodes.*"
+      - "compute.licenses.*"
+      - "compute.projects.get"
+      - "compute.regionOperations.get"
+      - "compute.regionOperations.list"
+      - "compute.regions.*"
+      - "compute.resourcePolicies.*"
+      - "compute.snapshots.*"
+      - "compute.zoneOperations.get"
+      - "compute.zoneOperations.list"
+      - "compute.zones.*"
+      - "resourcemanager.projects.get"
+      - "resourcemanager.projects.list"
+      - "serviceusage.quotas.get"
+      - "serviceusage.services.get"
+      - "serviceusage.services.list"
+      - "iam.serviceAccounts.actAs"
+      - "iam.serviceAccounts.get"
+      - "iam.serviceAccounts.list"
+      - "resourcemanager.projects.get"
+      - "resourcemanager.projects.list"
```

Comment:
@RomanBednar how did you choose the permissions? CI does not pass with `Permission compute.instantSnapshots.* is not valid."`

(and no, /retest will really not help here).

Reply:
@jsafrane One of the criteria for CCO-251 is to remove usage or roles altogether, I just wanted to test if expanding current roles to permissions would work (the list is from GCE doc). Even though this would not change the permissions as it matches the roles we could remove the roles already if that's the requirement, leaving optimization for later.

I saw you opened another PR for un-reverting the change so I'll close this one, thanks.